README.md

```diff
@@ -37,12 +40,30 @@ By running `docker compose up -d` these steps take place:
 1. a [mongodb](./services/mongodb/) container is created with some initial data.
 2. the SciCat [backend v4](./services/backend/services/v4/) container is created and connected to (1).
 3. the SciCat [frontend](./services/frontend/) container is created and connected to (2).
-4. a reverse [proxy](./services/proxy) container is created and routes traffic to (2) and (3) through localhost subdomains, in the form: `http://${service}.localhost`. The frontend is available at simply `http://localhost`.
+4. a reverse [proxy](./services/proxy) container is created and routes traffic to (2) and (3) through localhost subdomains, in the form: `http://${service}.localhost`. The frontend is available at simply `http://localhost`. 
+5. You can explore the API endpoints through the Swagger UI which is available on `http://backend.localhost/explorer`
 
 ## Extra services and features
 
 Features and services can be enabled or configured by setting [docker compose env variables](https://docs.docker.com/compose/environment-variables/envvars-precedence/), using [docker compose profiles](https://docs.docker.com/compose/profiles/), modifying the [service-specific config](#service-specific-config) and adding [entrypoints](#entrypoints).
 
+Services that can be added are:
+* LDAP - authentication and authorization from an LDAP server
+* OIDC - authentication and authorization using an OIDC provider
+* ElasitcSearch - for better free text search in the metadata
+* JupyterHub - Adds an instance of JupyterHub which demonstrates ingestion and extraction of metadata using [pyscicat](https://scicatproject.github.io/pyscicat/).
+
+Services in their default config can be easily set up by running:
+
+```
+docker compose up -d {service_name}
+```
+e.g.
+```
+docker compose up -d jupyter 
+```
+will spin up the jupyter service and its related dependencies.
+
```

Comment:
I think this is specified in [this block](https://github.com/SciCatProject/scicatlive?tab=readme-ov-file#select-the-services)

Reply:
I really want to make this obvious, but we can delete it if necessary.

Reply:
I am a little more concerned here than before, because this block is more complex. For example this: 
`docker compose up -d jupyter ` will not work, because jupyter also needs the profile to be set. So maybe we can limit it to: 
Services in their default config can be easily set up by running:

```
docker compose up -d {service_name}
```
and then point to the other section?

Reply:
Ok I can see it is a bit more involved from the block below ( though I managed to get it working from the default config). I will have a look at maybe restructuring the informartion in the Extra Services section, as I think the diagram should come higher, and the table maybe lower...